### DISCUSSION TITLE
How to replace female deck connectors on flowdeck

### DISCUSSION THREAD

**Discussion by akshaykumarsunkara** [bitcraze/discussions#1491] [Q&A]
Hey guys, as you can see the female deck connectors on this flow deck is all messed up.

I wanted to replace this with ones that I just bought - I also have a soldering workstation.

Just wondering if you guys could give a rundown of how to do this since I’m scared of ruining my flow deck. 

It would be great if you guys could link some YT videos I could refer to maybe and give some tips to not mess up the board.

thanks a lot!
![F40A8689-E6D5-4528-BA05-49A7B58E3818](https://github.com/user-attachments/assets/33b1e6c3-eb9d-4289-8085-f951cb48a99b)
![A5A04370-6051-42CC-A6A4-A92B9309AE56](https://github.com/user-attachments/assets/70fefef7-33e4-4121-a6e2-7d3632414b62)

**Comment by knmcguire** [comment]
We don't have a tutorial to solder the female connectors back on, but we do have one to solder pins directly to the deck, which might be useful for you: https://www.bitcraze.io/documentation/tutorials/solder-deck-pins/
